lib/matplotlib/backend_managers.py

```diff
@@ -56,14 +56,13 @@ class ToolManager(object):
         `LockDraw` object to know if the message is available to write
     """
 
-    def __init__(self, canvas):
+    def __init__(self, manager):
```

Comment:
I feel a bit odd about this, we create the ``ToolManager`` based on a ``FigureManager``, yet we use getters/setters work directly on the figure, in fact I don't ever see us use the manager passed in here, so it seems pointless as far as I can see, and makes the API more convoluted and difficult to learn.  Especially for embedded canvas widgets.  Does this bear relation to your multi-figure manager?  In which case can you document the use case of passing in a FigureManager here instead of a figure or canvas?